crates/core/src/lib.rs

```diff
@@ -92,8 +92,13 @@ pub type HostEnvValues = WitMap<String>;
 pub struct OtelConfig {
     /// OTEL_TRACES_EXPORTER https://opentelemetry.io/docs/concepts/sdk-configuration/general-sdk-configuration/#otel_traces_exporter
     pub traces_exporter: Option<String>,
+    /// OTEL_METRICS_EXPORTER https://opentelemetry.io/docs/concepts/sdk-configuration/general-sdk-configuration/#otel_metrics_exporter
+    pub metrics_exporter: Option<String>,
```

Comment:
Same comment as in `main`, but I don't think we need to add support for multiple exporters at this time